packages/amplication-client/src/Project/ProjectListItem.tsx

```diff
@@ -24,7 +24,7 @@ export const ProjectListItem = ({ project, workspaceId }: Props) => {
 
   return (
     <ListItem showDefaultActionIcon={true} onClick={handleClick}>
-      <Link to={`/${workspaceId}/${project.id}`}>
+      <Link to={{}}>
```

Comment:
Since we are not doing anything specific in the handleClick button - it is better to keep the link on the to property and remove the hadnler
this way, we keep the native behaviour of the browser - for exmaple, the user can use right click and choose "open in new tab"

Reply:
Got it, thank you.